src/gui/Src/Gui/CallStackView.cpp

```diff
@@ -49,6 +49,21 @@ void CallStackView::setupContextMenu()
     {
         return !getCellContent(getInitialSelection(), ColFrom).isEmpty() && isSelectionValid();
     });
+    QAction* loadSymbols = makeAction(DIcon("pdb"), tr("Download Symbols for Module"), SLOT(loadSymbolsSlot()));
+    mMenuBuilder->addAction(loadSymbols, [this](QMenu*)
+    {
+       return isSelectionValid();
+    });
+    QAction* loadSymbolsSingleThread = makeAction(DIcon("pdb"), tr("Download Symbols for All Modules (This Thread)"), SLOT(loadSymbolsSingleThreadSlot()));
+    mMenuBuilder->addAction(loadSymbolsSingleThread, [this](QMenu*)
+    {
+        return isSelectionValid();
+    });
+    QAction* loadSymbolsAllThreads = makeAction(DIcon("pdb"), tr("Download Symbols for All Modules (All Threads)"), SLOT(loadSymbolsAllThreadsSlot()));

```

Comment:
I don't know how useful this is, but it felt natural to include it along with the command which loads symbols for entries of a single thread. If it doesn't look useful, I'm happy with not including it.

Reply:
Yeah, I honestly think that downloading them for all threads is unnecessary. Probably the user will just download symbols for all modules in that case.

I'm also thinking to just have a single 'download symbols for this thread', because most likely that's what a user wants regardless. This also mirrors WinDbg's `k` command, which is pretty nice in this regard.

Reply:
That's fine. I've now removed the single-entry and all-thread commands and left only the single-thread command. I would've sparsely used the single-entry command, but it may not be all that useful in general. If others request it, it can always be re-added.